R/server-helpers.R

```diff
@@ -0,0 +1,28 @@
+showIntroModal <- function() {
+  showModal(
+    modalDialog(
+      title = "Welcome to stanpumpR",
+      p(
+        "stanpumpR, derived from the original STANPUMP program developed at
+        Stanford University,  performs pharmacokinetic simulations
+        based on mathematical models published in the peer-reviewed
+        literature. stanpumpR is intended to help clinicians and investigators
+        better understand the mathematical implications of published models.
+        stanpumpR is only an advisory program. How these models are applied to
+        individual patients is a matter of clinical judgment by the health care
+        provider."
+      ),
+      tags$button(
+        type = "button",
+        class = "btn btn-warning",
+        `data-dismiss` = "modal",
+        "OK"
+      ),
+      footer = NULL,
+      easyClose = TRUE,
+      fade = TRUE,
+      size = "s"
+    )
+  )
+}
+
```

Comment:
Because I was overwhelmed by the size of the server file, I tried moving chunks of code that didn't have to belong in the server outside, like `showIntroModal()` and `getReferenceTime()`